pkgs/build-support/kernel/modules-closure.sh

```diff
@@ -87,6 +87,15 @@ for module in $(< ~-/closure); do
     done || :
 done
 
+for path in $extraFirmwarePaths; do
+    mkdir -p $(dirname $out/lib/firmware/$path)
+    for name in "$path" "$path.xz" "$path.zst" ""; do
+        if cp -v --parents --no-preserve=mode lib/firmware/$name "$out" 2>/dev/null; then
```

Comment:
What's the deal with combining `-v` with `2>/dev/null`?

Reply:
It's pretty much a copy/paste of the `cp` commands in the same file. Interestingly `cp -v` doesn't print to stderr, so `2>/dev/null` doesn't wipe out the increased verbosity.